src/pages/Friend/FriendPage.jsx

```diff
@@ -0,0 +1,258 @@
+import { useState, useEffect, useRef } from 'react';
+import { arrayChop } from '../../components/common/CarouselComponent';
+import CarouselComponent from '../../components/common/CarouselComponent';
+import BottomSheetComponent from '../../components/common/BottomSheetComponent';
+import Button from '../../components/common/Button';
+import VerticalCard from '../../components/Friend/VerticalCard';
+import HorizontalCard from '../../components/Friend/HorizontalCard';
+import { ReactComponent as IcnUnion } from '../../assets/Friend/icn_union.svg';
+
+// const data = null;
+const data = [
+  {
+    nickname: '닉네임1',
+    birthday: '3월 18일',
+  },
+  {
+    nickname: '닉네임2',
+    birthday: '5월 14일',
+  },
+  {
+    nickname: '닉네임999999999999999999999999',
+    birthday: '12월 21일',
+  },
+];
+
+const FriendPage = () => {
+  const chopedDataList = data && arrayChop(data, 2);
+  const [bottomSheetShow, setBottomSheetShow] = useState(false);
+  const inputRef = useRef(null);
+  const [email, setEmail] = useState('');
+
+  // 바텀시트 렌더링 시 input 포커스
+  useEffect(() => {
+    inputRef.current?.focus();
+  });
+
+  return (
+    <SLayout>
+      <SH1 as='header'>친구</SH1>
+      {data && (
+        <SSection>
+          <ST1>나에게 선물한 친구</ST1>
+          <CarouselComponent pageLength={chopedDataList.length} pageWidth={335}>
+            {chopedDataList.map((item, index) => (
+              <SPageContainer key={index}>
+                <VerticalCard data={item[0]} />
+                {index === chopedDataList.length - 1 &&
+                data.length % 2 !== 0 ? (
+                  <div style={{ visibility: 'hidden' }}>
+                    <VerticalCard />
+                  </div>
+                ) : (
+                  <VerticalCard data={item[1]} />
+                )}
+              </SPageContainer>
+            ))}
+          </CarouselComponent>
+        </SSection>
+      )}
+      <SFriendSection>
+        <STopContainer>
+          <ST1>내 친구</ST1>
+          <SAddBtn
+            onClick={() => {
+              setBottomSheetShow(true);
+            }}
+          >
+            <IcnUnion />
+            친구 추가
+          </SAddBtn>
+        </STopContainer>
+        {data ? (
+          <SUl>
+            {data.map((item, index) => (
+              <li key={index}>
+                <HorizontalCard data={item} />
+              </li>
+            ))}
+          </SUl>
+        ) : (
+          <SGuideText>친구에게 초대 메시지를 보내보세요!</SGuideText>
+        )}
+      </SFriendSection>
+      {bottomSheetShow && (
+        <BottomSheetComponent
+          closeButton='cross'
+          setBottomSheetShow={setBottomSheetShow}
+        >
+          <SBottomSheetContainer>
+            <STextContainer>
+              <SH3>친구 추가</SH3>
+              <SB3>친구의 이메일 주소를 입력해 주세요</SB3>
+            </STextContainer>
+            <SForm
+              onSubmit={(event) => {
+                event.preventDefault();
+                setEmail('');
+              }}
```

Comment:
```
const handleFormSubmit = (event) => {
    event.preventDefault();
    setEmail('');
};
```
```suggestion
              onSubmit={handleFormSubmit}
```

굳이굳이 바꾸자면 이벤트 핸들러 등록할 때 함수 따로 작성하기 정도가 있을 거 같아요